example/example-model/src/main/java/uk/gov/gchq/palisade/example/client/ExampleSimpleClient.java

```diff
@@ -165,8 +169,11 @@ private void initialiseServices() {
         return super.read(filename, RESOURCE_TYPE, userId, justification);
     }
 
-    @Override
-    protected Serialiser<ExampleObj> createSerialiser() {
-        return new ExampleObjSerialiser();
+    private ParentResource getParent(final String fileURL) {
+        if (fileURL.isEmpty()) {
+            return new SystemResource().id("/");
+        }
+        String parentURL = fileURL.substring(0, fileURL.lastIndexOf("/"));
```

Comment:
Thinking about possible Windows compat. please can this use FileSystem.getSeparator()

Reply:
This might require a fair bit of extra work as the file system is not in that class